src/main/java/dev/aura/bungeechat/util/DummyPlayer.java

```diff
@@ -17,8 +10,12 @@
 import net.md_5.bungee.api.connection.PendingConnection;
 import net.md_5.bungee.api.connection.ProxiedPlayer;
 import net.md_5.bungee.api.connection.Server;
+import net.md_5.bungee.api.event.ServerConnectEvent;
 import net.md_5.bungee.api.score.Scoreboard;
 
+import java.net.InetSocketAddress;
+import java.util.*;
+
```

Comment:
Imports should be formated correctly. Just run `./gradlew spotlessApply` to fix imports.